ecs-cli/modules/clients/aws/ecs/client.go

```diff
@@ -368,23 +368,8 @@ func (c *ecsClient) DescribeTasks(taskArns []*string) ([]*ecs.Task, error) {
 }
 
 // RunTask issues a run task request for the input task definition
-func (c *ecsClient) RunTask(taskDefinition, group string, count int, networkConfig *ecs.NetworkConfiguration, launchType string) (*ecs.RunTaskOutput, error) {
-	runTaskInput := &ecs.RunTaskInput{
-		Cluster:        aws.String(c.config.Cluster),
-		TaskDefinition: aws.String(taskDefinition),
-		Group:          aws.String(group),
-		Count:          aws.Int64(int64(count)),
-	}
-
-	if networkConfig != nil {
-		runTaskInput.NetworkConfiguration = networkConfig
-	}
-
-	if launchType != "" {
-		runTaskInput.LaunchType = aws.String(launchType)
-	}
-
-	resp, err := c.client.RunTask(runTaskInput)
+func (c *ecsClient) RunTask(taskDefinition string, input *ecs.RunTaskInput) (*ecs.RunTaskOutput, error) {
```

Comment:
since `input` already includes the task definition, why do we need it as a separate param here?